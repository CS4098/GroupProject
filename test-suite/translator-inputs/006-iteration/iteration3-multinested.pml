process iteNest {

    action a {}
    iteration {
        action b {}
        iteration  {
            action c {}
            iteration {
                action d {}
            }
        }
    }
    action d {}
}
