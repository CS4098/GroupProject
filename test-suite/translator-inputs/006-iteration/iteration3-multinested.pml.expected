
active proctype iteNest()
{
	int count__;
    for (count__ : 1 .. 4) {
        int count___;
        for (count___ : 1 .. 4) {
            int count____;
            for (count____ : 1 .. 4) {
            }
        }
    }
}
